src/psyclone/tests/psyir/transformations/intrinsics/dotproduct2code_trans_test.py

```diff
@@ -352,9 +348,12 @@ def test_apply_unknown_dims(tmpdir, fortran_reader, fortran_writer):
     check that the correct precision is used.
 
     '''
+    if Compile.TEST_COMPILE:
+        pytest.skip("issue #1347, type declaration is being written in the "
```

Comment:
That's weird. I thought the backend always generated parameter declarations first. Oh well.

Reply:
Not yet!